File: src/main/java/com/model/Statistics.java
package com.model;

import com.fasterxml.jackson.annotation.JsonFormat;

import java.math.BigDecimal;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class Statistics {
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private BigDecimal sum;
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private BigDecimal avg;
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private BigDecimal max;
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private BigDecimal min;
    @JsonFormat(shape = JsonFormat.Shape.NUMBER_INT)
    private long count;
}

File: src/main/kotlin/com/model/Statistics.kt
package com.model

import com.fasterxml.jackson.annotation.JsonFormat
import lombok.Builder
import lombok.Data
import java.math.BigDecimal

@Data
data class Statistics(
    @get:JsonFormat(shape = JsonFormat.Shape.STRING) val sum: BigDecimal,
    @get:JsonFormat(shape = JsonFormat.Shape.STRING) val avg: BigDecimal,
    @get:JsonFormat(shape = JsonFormat.Shape.STRING) val max:BigDecimal,
    @get:JsonFormat(shape = JsonFormat.Shape.STRING) val min:BigDecimal,
    @get:JsonFormat(shape = JsonFormat.Shape.NUMBER_INT) val count:Long
)